source/adminguide/veeam_plugin.rst

```diff
@@ -0,0 +1,152 @@
+#. Setup your 'Backup Repositories' in the Veeam Backup & Replication Console.  Remember that you will likely want a different
+   target in each zone.
+
+Creating Template jobs
+----------------------
+
+#. As noted above, a dummy VM tag is required in order to create template jobs which don't contain any VMs.  This is done via vCenter
+   by navigating to the 'Tags and Custom Attributes' section, and first creating a category (if you dont already have a suitable one).
+
+   |BnR-DummyTagCategory.jpg|
+
+#. Then, create a dummy tag in that category.  Be sure to make it clear that it is not to be used anywhere.
+
+   |BnR-CreateDummyTag.jpg|
+
+#. Now create the template job in Veeam Backup and Replication Manager. using the New Backup Job (Virtual Machine) wizard.
+
+   #. Give the job a name that describes what the job does ie *template_job_zone1_default* or *template_daily_job-14_kept*
+      (the end user will not see this name).
+   #. In the Virtual Machines section of the wizard, click 'Add' and select the 'VMs and Tags' filter (top right of the 
+      'Add Objects' dialog box). And then select your dummy tag and click on Add.
+
+      |BnR-VMsandTags.jpg|
+
+   #. In the Storage section is the correct Backup repository for the zone and number of restore points. (note there are a number
+      of other advanaced options which can be set, these are transparent to CloudStack. CloudStack will clone this job 'as-is' including
+      all advanced settings.  However changing these settings will only effect NEW jobs created from the template, existing jobs will be
+      unchanged.
+   
+   #. The same is true for the Guest Processing section. 
+
+   #. In the Schedule section you, if you are creating an 'SLA' based backup template, you would set the job to run automatically and
+      select 'Periodically every' 24hrs and then in the 'Schedule' dialog set the hours in which the job is allowed to run.  This allows
+      Veeam to choose the best time to run the backup within a given window.  If you are creating a template for adhoc/scheduled backups,
+      do not tick 'Run the job automatically' as CloudStack will trigger jobs as and when required.
+
+      |BnR-backupschedule.jpg|
+
+   #. Finally, save the job.
+
+
+Connecting CloudStack to Veeam
+-------------------------------
+
+Once Veeam is configured with SSH enabled and at least one template job, we can connect CloudStack to your Veeam server.
+
+To do this, you simply update the global settings listed below:
+
+Plug-in specific settings:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+(all settings can be global or per-zone)
+
+.. cssclass:: table-striped table-bordered table-hover
+
+==================================== ========================
+Configuration                         Description
+==================================== ========================
+backup.plugin.veeam.url              Veeam B&R server URL. Default: http://<VEEAM_SERVER_IP>:9399/api/
```

Comment:
i think the default port has changed with the latest veeam: http://<VEEAM_SERVER_IP>:9398/api/

Reply:
updated.